synchronization/src/cyface/java/de/cyface/synchronization/JsonMapper.java

```diff
@@ -28,26 +31,48 @@
 
     /**
      * Transforms measurement points available as {@link JSONObject}s to {@link ContentProviderOperation}s
-     * which mark the points as synced when executed.
+     * which delete the points when executed.
      *
      * @param measurementSlice The measurement slice which holds the points
      * @param authority The authority to access the data in the database.
-     * @return A {@link Collection<ContentProviderOperation>}s which marks the points as synced.
+     * @return A {@link Collection<ContentProviderOperation>}s which deletes the points.
      * @throws SynchronisationException When the data could not be mapped.
      */
-    Collection<ContentProviderOperation> buildMarkSyncedOperation(JSONObject measurementSlice, String authority)
+    Collection<ContentProviderOperation> buildDeleteDataPointsOperation(JSONObject measurementSlice, String authority)
             throws SynchronisationException;
 
     /**
-     * Transforms measurement points available as {@link JSONObject}s to {@link ContentProviderOperation}s
-     * which delete the points when executed.
-     * TODO: We'll delete this method again when we implemented the Cyface Byte Code sync.
+     * A delegate which offers methods to build database operations for {@link JsonMapper}s of measurement data points.
      *
-     * @param measurementSlice The measurement slice which holds the points
-     * @param authority The authority to access the data in the database.
-     * @return A {@link Collection<ContentProviderOperation>}s which deletes the points.
-     * @throws SynchronisationException When the data could not be mapped.
+     * @author Armin Schnabel
+     * @version 1.0.0
+     * @since 2.4.0
      */
-    Collection<ContentProviderOperation> buildDeleteOperation(JSONObject measurementSlice, String authority)
-            throws SynchronisationException;
+    class JsonDatabaseMapper {
+        public Collection<ContentProviderOperation> buildDeleteOperation(final JSONObject measurementSlice,
```

Comment:
Mach doch aus dem Rückgabetyp direkt eine ArrayList. Die applyBatch Methode ist die einzige die eine Collection von ContentProviderOperation annehmen kann und die will seltsamerweise eine ArrayList haben. Dann sparst du dir oben die Rückgabe von hier nochmal in eine ArrayList zu wrappen.